float64[3] pos
float64[3] RPY
float64 vel
float64 acc
bool wait_flag